### ISSUE TITLE
scrpy.exe does not open at all in windows 11.

### ISSUE THREAD

**Issue by Danielmtdc16** [Genymobile/scrcpy#3753]
After connecting the device through the USB cable (with debugging enabled), and running scrpy.exe, the prompt opens and closes quickly.

**Comment by mcDandy** [comment]
Try opening a command prompt (where you have scrcpy.exe) and run scrpy.exe from there

**Comment by rom1v** [comment]
Or run `scrcpy-console.bat`, it won't close on error.

**Comment by Danielmtdc16** [comment]
I went to test today again through the file. When I opened it, it worked normally. Thanks.

**Comment by NEOgeek402** [comment]
Yeah does not work at all in any way on any device I have. I tried on 7 systems at the hotel not one worked. All crash instantly no matter what you do. though the adb.exe will run in the background all day. I do not trust that at all. IMO Githubs very aggravating and rarely produces anything worth wild mostly virus filled BS I am assuming.

**Comment by rom1v** [comment]
> All crash instantly no matter what you do. 

Run `scrcpy-console.bat` to get the error message.

> adb.exe will run in the background all day

https://github.com/Genymobile/scrcpy/issues/205

> IMO Githubs very aggravating and rarely produces anything worth wild mostly virus filled BS I am assuming.

Github does not produce scrcpy (it just hosts the repo), I do. But thank you for your kind words. 🙃

**Comment by NEOgeek402** [comment]
Tried that it crashes all the same. I don't trust anyone, I am aware of how GitHub works just not a fan of people. Things rarely work as intended and people like to be dirty. Why would I not assume the same of you not personal just reality

**Comment by dxomg** [comment]
> Tried that it crashes all the same. I don't trust anyone, I am aware of how GitHub works just not a fan of people. Things rarely work as intended and people like to be dirty. Why would I not assume the same of you not personal just reality

... Its an opensource project, means even you can modify & check the code accordingly to Apache-2.0 license, also by it being an opensource project you yourself could check if it has any malicious code (which it dosent).

**Comment by rom1v** [comment]
> Tried that it crashes all the same.

That does not help. If you run `scrcpy-console.bat`, it will keep the console open on error. What is the console output?

Alternatively, run `open_a_terminal_here.bat` and run scrcpy.

**Comment by princeofdream** [comment]
scrcpy can not get adb device in win 11

```shell
PS D:\> adb device
adb.exe: unknown command device
PS D:\> adb devices
List of devices attached
11713dcd        device

PS D:\> scrcpy-console.bat
scrcpy 3.1 <https://github.com/Genymobile/scrcpy>
ERROR: Could not list ADB devices
ERROR: Server connection failed
Press Enter to continue...
```

**Comment by rom1v** [comment]
#3386 #3534
